code/computational_geometry/src/sphere_tetrahedron_intersection/sphere_tetrahedron_intersection.cpp

```diff
@@ -0,0 +1,1237 @@
+/*
+Analytical calculations for sphere-tetrahedron intersections...
+These calculations use a lot of vector math, so I had to include my vector class.
+Vector class ends at around line 900, and intersection codes start.
+
+ - RedBlight
+*/
+
+#ifndef LU_VECTOR_INCLUDED
+#define LU_VECTOR_INCLUDED
+
+#include <sstream>
+#include <string>
+#include <complex>
+#include <vector>
+#include <iostream>
+
+namespace LUV
+{
+	// LuVector.hpp
+	
+	using namespace std;
```

Comment:
Please please please avoid this at ALL costs.

Reply:
I thought using it in a namespace would limit it's scope and there wouldn't be any problems but a quick search proved otherwise. I'll not use it anymore.